routes/offlinetraces.js

```diff
@@ -8,6 +8,32 @@ module.exports = function (kafkaConfig) {
 
     var offlinetraces = require('../lib/offlinetraces')(kafkaConfig);
 
+    /**
+     * @api {get} /api/offlinetraces/:activityId/kahoot Returns the Kahoot offlinetraces that has the given id.
+     * @apiName GetOflinetraces
+     * @apiGroup Oflinetraces
+     *
+     * @apiParam {String} id The Analysis id
+     *
+     * @apiSuccess(200) Success.
+     *
+     * @apiSuccessExample Success-Response:
```

Comment:
Documentation say "Returns the Kahoot offlinetraces" but this Response is not like a group of traces....

Reply:
I see

Reply:
This is because it resturns a mongo document with meta-information about the uploaded Kahoot file, the file is not returned